doc/adr/0011-tsp-analysis-api.md

```diff
@@ -8,124 +8,124 @@ Proposed
 
 ## Context
 
-The trace viewer currently is able to visualize trace data provided by a trace server over the trace server protocol (TSP). The Trace Compass server has some built-in analysis view for that. However, it is not possible to side-load analysis and visualization descriptions over the TSP so that end-user can provide some custom views. The Trace Compass supports loading of data-driven analysis and views, e.g. XML driven views or in-and-out anlysis of the Trace Compass incubator project. In the Eclispe-based Trace Compass application, there exists UI primitive to load e.g. XML files or configure custom analysis. While the Trace Compass server has the capablility to understand these defintions, there is no way to side-load this definition over the TSP. This ADR will propose an analysis service within the TSP to facilitate these custom analysis.
+The trace viewer currently is able to visualize trace data provided by a trace server over the trace server protocol (TSP). The Trace Compass server has some built-in analysis view for that. However, it is not possible to side-load analysis and visualization descriptions over the TSP so that end-user can provide some custom views. The Trace Compass supports loading of data-driven analysis and views, e.g. XML driven views or in-and-out anlysis of the Trace Compass incubator project. In the Eclispe-based Trace Compass application, there exists UI primitive to load e.g. XML files or configure custom analysis. While the Trace Compass server has the capablility to understand these defintions, there is no way to side-load this definition over the TSP. This ADR will propose a configuration service within the TSP to facilitate these custom analysis. The proposed configuration service can be use to configure other server specific customization, e.g. custom trace parsers.
 
-### Global analysis service
+### Global configuration service
 
-An analysis service for managing global analyses will be introduced. Global in this context means that the analysis definitions will be handled on the application level. 
+An analysis service for managing global configurations will be introduced. Global in this context means that the configuration definitions will be handled on the application level. 
 
-    GET /analysis/
-        returns a list of analysis type descriptors: typeId, name, description, expected input parameter list ("path" for file path)
-    GET /analysis/types
-        returns a map typeId -> list of analysis descriptor of existing analyses on server: analysisId, name, description, parameters
-    GET /analysis/types/{typeId}
+    GET /config/
+        returns a list of configuration source types: typeId, name, description, expected input parameter list ("path" for file path)
```

Comment:
input parameter -> query parameter keys

Reply:
missing scope, according to the descriptor

Reply:
done